packages/legacy/src/queries/tree.ts

```diff
@@ -6,6 +6,7 @@ import { SourceInventoryProvider } from './types';
 import { IInventoryHostTree, InventoryTree, InventoryTreeType } from './types/tree.types';
 import { consoleFetchJSON } from '@openshift-console/dynamic-plugin-sdk';
 import { ProviderType } from '../common/constants';
+import { usePollingContext } from 'legacy/src/common/context';
```

Comment:
```suggestion
import { usePollingContext } from '../common/context';
```

no reason really, just to align with line 8